def check_solution(group, task, variant, code):
    return False, "Error occured."
